File: branches/Medusa/Source/Fantasy.Jobs/CallTemplateParameter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;

namespace Fantasy.Jobs
{
    [XSerializable("parameters", NamespaceUri = Consts.XNamespaceURI)]
    public class CallTemplateParameter : IConditionalObject
    {
        [XAttribute("itemCategory")]
        public string ItemCategory { get; set; }

        [XAttribute("include")]
        public string Include { get; set; }

        [XAttribute("condition")]
        public string Condition { get; set; }

        [XAttribute("propertyName")]
        public string PropertyName { get; set; }

        [XAttribute("value")]
        public string Value { get; set; }
    }
}

File: AutoConverted/Fantasy.Jobs/CallTemplateParameter.java
﻿package Fantasy.Jobs;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[XSerializable("parameters", NamespaceUri = Consts.XNamespaceURI)]
public class CallTemplateParameter implements IConditionalObject
{
//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("itemCategory")]
	public String ItemCategory = null;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("include")]
	public String Include = null;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("condition")]
	private String _condition = null;
	private String getCondition()
	{
		return this._condition;
	}
	private void setCondition(String value)
	{
		this._condition = value;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("propertyName")]
	public String PropertyName = null;
//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[XAttribute("value")]
	public String Value = null;
}